### PR TITLE
Added turing.com

### PR DESCRIPTION
**Pull request by KapilKumar7**
📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.
- [ ] I am an employee of the company mentioned and confirm all included details are correct.
- [ ] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [ ] The company directly hires employees. No bootcamps / freelance sites / etc
- [ ] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [ ] A [company profile](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md) is included - This is __required__ for new additions.
- [ ] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [ ] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [ ] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/tree/main/.github/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/tree/main/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: company-profiles/turing.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Remote status
+
+100 % remote jobs .
+
+## Region
+
+Worldwide
+
+## Company technologies
```
Comment (dougaitken) [inline company-profiles/turing.md:22]: These aren't technologies so either remove this section completely or edit to match programming languages or similar.

File: company-profiles/turing.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ 
+We make the remote hiring journey easy and rewarding for both companies and developers. With Turing, companies can hire pre-vetted, Silicon Valley-caliber remote software talent across 100+ skills in 3-5 days. We also democratize opportunities for remote developers from around the world by offering them high-quality software jobs with top US firms.
+
+ 
+Our Intelligent Talent Cloud uses AI to source, vet, match, and manage over 2 million developers worldwide. This, in turn, helps organizations save valuable time and resources as they build their dream engineering team in a matter of days.
+
+
+## Remote status
+
+100 % remote jobs .
```
Comment (dougaitken) [inline company-profiles/turing.md:16]: Two unneeded spaces. Please change to this:

```
100% remote jobs.
```

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Comment by KapilKumar7** [comment]
I have removed the whitespaces and Removed the technology section as well.  Please let me know if there is something else where I need to make changes.

File: company-profiles/turing.md (hunk 1 above)
Comment (KapilKumar7) [reply company-profiles/turing.md]: I have removed the technologies section.

File: company-profiles/turing.md (hunk 2 above)
Comment (KapilKumar7) [reply company-profiles/turing.md:11]: Done.

**Review by dougaitken — APPROVED** [review]

**Pull request merged**